llama_stack/templates/together/run-with-safety.yaml

```diff
@@ -99,46 +99,91 @@ metadata_store:
   type: sqlite
   db_path: ${env.SQLITE_STORE_DIR:~/.llama/distributions/together}/registry.db
 models:
+- metadata: {}
+  model_id: meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo
+  provider_id: together
+  provider_model_id: meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo
+  model_type: llm
 - metadata: {}
   model_id: meta-llama/Llama-3.1-8B-Instruct
   provider_id: together
   provider_model_id: meta-llama/Meta-Llama-3.1-8B-Instruct-Turbo
   model_type: llm
+- metadata: {}
+  model_id: meta-llama/Meta-Llama-3.1-70B-Instruct-Turbo
```

Comment:
n00b qs: why we have the `-Turbo` suffix here, but not in `meta-llama/Llama-3.1-8B-Instruct`

Reply:
@yanxi0830 this is the ID that Together uses. they have some models which get served faster (quantized)